src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -8,9 +8,11 @@ import {
   TouchableOpacity,
   View
 } from 'react-native';
+
+import MemoDomainProfilePicture from '../../../assets/icon/DomainProfilePictureEmptyState';
+import TopicsProfilePictureEmptyState from '../../../assets/icon/TopicsProfilePictureEmptyState';
 import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements for `colors` and `fonts` have been moved down. This change doesn't seem to affect the functionality of the code, but it's generally a good practice to keep all import statements together at the top of the file for better readability and maintainability.

```diff
- import {colors} from '../../../utils/colors';
- import {fonts} from '../../../utils/fonts';
  import MemoDomainProfilePicture from '../../../assets/icon/DomainProfilePictureEmptyState';
+ import {colors} from '../../../utils/colors';
+ import {fonts} from '../../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->